README.md

```diff
@@ -29,7 +29,7 @@ Your answer...
 In a Rails application, how is the router related to controller actions?  
 
 ```text
-Your answer...
+The actions or steps in the controller file are dictated by the routes file. Each controller action has a specific route from the router.
 ```
```

Comment:
The router does direct the user to particular controller actions. But more specifically, each route in a router links a particular HTTP request to a particular controller action .